### ISSUE TITLE
slick.jdbc.PostgresProfile.api._ will be conflict with slickpg.ExPostgresProfile.api._

### ISSUE THREAD

**Issue by asdfping** [tminglei/slick-pg#468]
if you import `com.github.tminglei.slickpg.ExPostgresProfile.api._`，
and import `slick.jdbc.PostgresProfile.api._` meanwhile，you will compile error.

if you import `com.github.tminglei.slickpg.ExPostgresProfile.api.Over`，
and import `slick.jdbc.PostgresProfile.api._` meanwhile , you will get exception `slick.SlickException: Unexpected node WindowFuncExpr`.

So, you should  import `com.github.tminglei.slickpg.ExPostgresProfile.api._` instead of `slick.jdbc.PostgresProfile.api._` if you want to use slick-pg

**Comment by tminglei** [comment]
Yes, `ExPostgresProfile` extended `PostgresProfile`, and `PostgresProfile` didn't know `com.github.tminglei.slickpg.ExPostgresProfile.api.Over`.

**Comment by charlietsai** [comment]
I think this might be related, but I ran into issues with `ExPostgresProfile` and `slick.jdbc.JdbcProfile` in slick codegen. The implicits were not being used by the generated code with the default generated :
```scala
object MyTable extends {
  val profile = MySlickPgPostgresProfile
} with MyTable

trait MyTable {
  val profile: slick.jdbc.JdbcProfile
  import profile.api._
  ...
}
```
wasn't working as expected, but replacing `slick.jdbc.JdbcProfile` with my custom profile resolved the problem.

This can be added to a custom code generator using:
```scala
      override def packageCode(profile: String, pkg: String, container: String, parentType: Option[String]): String = {
        val defaultProfileName = "slick.jdbc.JdbcProfile"
        super
          .packageCode(profile, pkg, container, parentType)
          .replace(defaultProfileName, jdbcProfile match {
            case slickPgProfile: ExPostgresProfile => profileName
            case _                                 => defaultProfileName
          })
      }
```
